Assets/Scripts/NetworkManager.cs

```diff
@@ -27,7 +27,9 @@ void Start()
                 Instance = this;
                 Object.DontDestroyOnLoad(gameObject);
                 PhotonNetwork.automaticallySyncScene = true;
-                if (!SceneManager.GetSceneByName("StartMenu").isLoaded)

```

Comment:
must have missed this in my tests. looks like some "trying to program at midnight" boolean logic. Good catch. will fix and report back